docs/command-line-reference.rst

```diff
@@ -5,54 +5,49 @@ Command Line Reference
 ######################
 
 While most of zenpacklib's functionality is as a Python module to be used as a
-library for helping build ZenPacks, `zenpacklib.py` also acts as a command line
+library for helping build ZenPacks, `zenpacklib` also acts as a command line
 script to perform some useful actions.
 
-You can choose to make `zenpacklib.py` executable then execute it as as script.
+The `zenpacklib` script can be run from the command line with: 
 
 .. code-block:: bash
+   `$ZENHOME/bin/zenpacklib` (usually `/opt/zenoss/bin/zenpacklib`)
```

Comment:
seeing this error when making html.  i think there's supposed to be a blank line between the code-block line and the actual code

```
/x/zenpacklib/docs/command-line-reference.rst:13: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 4 supplied.

.. code-block:: bash
   `$ZENHOME/bin/zenpacklib` (usually `/opt/zenoss/bin/zenpacklib`)
```